content/foxtrick.properties

```diff
@@ -503,6 +503,20 @@ module.U20LastMatch.SeniorPlayers.desc=Show last official U20 match in the Senio
 module.U20LastMatch.AllPlayers.desc=Show last official U20 match in the Senior/Youth/NT Players page.
 module.U20LastMatch.TransfersResults.desc=Show last official U20 match in the Transfers Search Results page.
 
+# Mercattrick
+module.MercattrickStats.desc=Display how many managers are interested in a transfer listed player.
+module.MercattrickStats.enableTLPage.desc=Enable on transfer search result page.
```

Comment:
Any plans to use this on other pages?

Reply:
I think I've added it to all the appropriate pages, so the answer is no, for now. :)